packages/lisk-transactions/src/base_transaction.ts

```diff
@@ -305,6 +306,66 @@ export abstract class BaseTransaction {
 		return createResponse(this.id, errors);
 	}
 
+	public addMultisignature(
+		store: StateStore,
+		signatureObject: SignatureObject,
```

Comment:
maybe destructure here?

Reply:
You mean the function parameters? It seems that in TS using destructuring assignment is more verbose than in JS should we reserve its use to methods with many parameters only?